Cargo.toml

```diff
@@ -23,7 +23,7 @@ subtle = { version = "2.2.1", default-features = false, features = ["i128"] }
 [features]
 default = ["bits", "std"]
 alloc = []
-bits = ["bitvec"]
+bits = ["bitvec", "ff_derive/bits"]
```

Comment:
We can't do this, because it means any downstream crate that adds the `bits` feature flag is also forced to depend on `ff_derive` due to how transitive feature flags work. This change is blocked on MSRV 1.60.0 (released this month), which introduces [weak dependency features](https://blog.rust-lang.org/2022/04/07/Rust-1.60.0.html#new-syntax-for-cargo-features).

Reply:
Aah excellent catch! I thought I had tried this out but I didn't think to try the not-using ff_derive one.

So ... is the solution just to drop this indirect enabling together with the other comment removal?

Reply:
Maybe it's to add another feature instead, since I think most use might be through `#[derive(ff::PrimeField)]`. A new feature should probably enable the `derive` and `bits` on `ff`, `derive_bits` or something?

Reply:
Yeah, that should be fine.

Reply:
Included the 1.60 version as a comment, to be unified hopefully at some point in time :)